code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -348,27 +348,30 @@
 	desc = "They say fear is the true mind killer, but stabbing them in the head works too. Honour compels you to not sheathe it once drawn."
 	sharpness = IS_SHARP
 	slot_flags = null
-	flags = HANDSLOW
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 
-/obj/item/weapon/nullrod/pitchfork
-	icon_state = "pitchfork0"
-	name = "unholy pitchfork"
-	w_class = WEIGHT_CLASS_NORMAL
-	desc = "Holding this makes you look absolutely devilish."
-	attack_verb = list("poked", "impaled", "pierced", "jabbed")
-	hitsound = 'sound/weapons/bladeslice.ogg'
-	sharpness = IS_SHARP
```

Comment:
Someone defined the pitchfork in the middle of the tribal_knife procs. So I moved it.